### PR TITLE
Translator python

### LINKED ISSUE #484: Translator Script in Python
# Aim
This Script helps to translate from one language to another

# Details
It uses Google Trans Library and is efficient in translating.
Requires : ```pip install googletrans==3.1.0a0```

I would Like to work on this issue,
Please assign this issue to me as part of GSSOC-21.
Thanks

### PR DESCRIPTION
**Pull request by NEERAJAP2001**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Fixes #484 

Replace `issue_no` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by antrikshmisri — CHANGES_REQUESTED** [review]
Instead of hardcoding the text and language take the inputs from the user. Also, I was not able to execute the code due to dependency issues. @kaustubhgupta @santushtisharma10 @tamanna-verma , can someone confirm if the code runs.

**Comment by NEERAJAP2001** [comment]
Alright, @antrikshmisri  changes coming up !! 

> Also, I was not able to execute the code due to dependency issues.

Please read the documentation, I've mentioned dependencies there!
Thanks

**Comment by antrikshmisri** [comment]
> Alright, @antrikshmisri changes coming up !!
> 
> > Also, I was not able to execute the code due to dependency issues.
> 
> Please read the documentation, I've mentioned dependencies there!
> Thanks

@NEERAJAP2001 I meant  I wasn't able to run the code with the mentioned dependencies on my local machine

File: Translator Script/Translation.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (kaustubhgupta) [inline Translator Script/Translation.py:1]: Remove the unnecessary blank lines and comments. It's 6 lines code and it will remain that only

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by NEERAJAP2001** [comment]
Screenshot (254)


@kaustubhgupta Works Fine!!

File: Translator Script/Translation.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# First Step : Go to cmd and type: pip install googletrans==3.1.0a0
```
Comment (kaustubhgupta) [inline Translator Script/Translation.py:1]: This first line is not needed as it is already mentioned in the README.md file dependency section

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Review by kaustubhgupta — APPROVED** [review]

**Comment by kaustubhgupta** [comment]
@santushtisharma10 approval awaited

**Review by antrikshmisri — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Comment by avinashkranjan** [comment]
@all-contributors please add @NEERAJAP2001  for code and documentation

**Pull request merged**